cluster/eks/composition.yaml

```diff
@@ -174,11 +174,12 @@ spec:
           credentials:
             source: Secret
             secretRef:
-              namespace: crossplane-system
```

Comment:
@jbw976 is this implicit namespace behavior documented somewhere? I looked on crossplane.io/docs but didn't find anything.

Reply:
can you clarify a bit more what you mean by "implicit namespace behavior" @prasek? the namespace is explicitly being set below in the patches, but perhaps I don't understand fully what you are implying

Reply:
my bad, I misread the patches while multi-tasking. 👍